2015-03-15 01:35:35 Error: [MissingActionException] Action CategoriesController::users() could not be found.
Exception Attributes: array (
  'controller' => 'CategoriesController',
  'action' => 'users',
)
Request URL: /cakephp/blog/categories/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(CategoriesController), Object(CakeRequest))
#2 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2015-03-15 01:35:55 Error: [NotFoundException] Invalid user
Request URL: /cakephp/blog/users/edit
Stack Trace:
#0 [internal function]: UsersController->edit()
#1 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2015-03-15 02:23:44 Error: [MissingControllerException] Controller class LoginController could not be found.
Exception Attributes: array (
  'class' => 'LoginController',
  'plugin' => NULL,
)
Request URL: /cakephp/blog/login
Stack Trace:
#0 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2015-03-15 02:23:44 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:23:44 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:24:21 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:24:21 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:24:25 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:24:25 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:29:46 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:29:46 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:29:48 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:29:48 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:29:50 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:29:50 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:29:51 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:29:51 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:23 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:23 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:24 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:24 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:24 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:24 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:24 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:24 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:24 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:24 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:25 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:25 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:25 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:25 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:25 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:25 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:25 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:25 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:25 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:25 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:25 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:25 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:26 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:26 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:26 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:26 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:26 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:26 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:27 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:27 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:27 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:27 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:27 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:27 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:27 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:27 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:30:31 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:30:31 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:35:06 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:35:06 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:35:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:35:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:35:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:35:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:35:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:35:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:05 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:05 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:05 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:05 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:06 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:06 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:06 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:06 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:06 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:06 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:06 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:06 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:06 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:06 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:07 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:07 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:07 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:07 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:07 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:07 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:07 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:07 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:07 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:07 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:07 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:07 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:09 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:09 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:09 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:09 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:09 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:09 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:09 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:09 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:10 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:10 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/users/add
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:38:17 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 44]
2015-03-15 02:38:17 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/posts
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 44, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 02:40:08 Error: Fatal Error (1): Class 'AuthComponent' not found in [/var/www/cakephp/blog/Controller/AppController.php, line 43]
2015-03-15 02:40:08 Error: [FatalErrorException] Class 'AuthComponent' not found
Request URL: /cakephp/blog/posts
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'AuthComp...', '/var/www/cakeph...', 43, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 03:17:01 Error: [MissingViewException] View file "Users/login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users/login.ctp',
)
Request URL: /cakephp/blog/users/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/View/View.php(470): View->_getViewFileName(NULL)
#1 /opt/cakephp/lib/Cake/Controller/Controller.php(960): View->render(NULL, NULL)
#2 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(200): Controller->render()
#3 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2015-03-15 03:20:53 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'User.username' in 'where clause'
Request URL: /cakephp/blog/users/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `User`.`...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `User`.`...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1106): DboSource->fetchAll('SELECT `User`.`...', false)
#4 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(User), Array)
#5 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#6 /opt/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(130): Model->find('first', Array)
#7 /opt/cakephp/lib/Cake/Controller/Component/Auth/FormAuthenticate.php(79): BaseAuthenticate->_findUser('jose', '123')
#8 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(762): FormAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#9 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(607): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#10 /var/www/cakephp/blog/Controller/UsersController.php(113): AuthComponent->login()
#11 [internal function]: UsersController->login()
#12 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#14 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#15 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#16 {main}
2015-03-15 03:38:27 Error: [MissingLayoutException] Layout file "Layouts/login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Layouts/login.ctp',
)
Request URL: /cakephp/blog/users/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/View/View.php(514): View->_getLayoutFileName('login')
#1 /opt/cakephp/lib/Cake/View/View.php(481): View->renderLayout('', 'login')
#2 /opt/cakephp/lib/Cake/Controller/Controller.php(960): View->render(NULL, NULL)
#3 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(200): Controller->render()
#4 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#5 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2015-03-15 03:41:11 Error: Fatal Error (4): syntax error, unexpected '}' in [/var/www/cakephp/blog/Controller/AppController.php, line 73]
2015-03-15 03:41:11 Error: [FatalErrorException] syntax error, unexpected '}'
Request URL: /cakephp/blog/users/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 73)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 73, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 73, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 03:41:38 Error: [MissingActionException] Action UsersController::posts() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'posts',
)
Request URL: /cakephp/blog/users/posts
Stack Trace:
#0 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#2 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2015-03-15 03:45:51 Error: Fatal Error (4): syntax error, unexpected '=', expecting ')' in [/var/www/cakephp/blog/View/Users/login.ctp, line 6]
2015-03-15 03:45:51 Error: [FatalErrorException] syntax error, unexpected '=', expecting ')'
Request URL: /cakephp/blog/users/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 6)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 6, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 6, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-15 16:42:54 Error: [MissingControllerException] Controller class CakephpController could not be found.
Exception Attributes: array (
  'class' => 'CakephpController',
  'plugin' => NULL,
)
Request URL: /cakephp/blog/cakephp/blog/posts
Stack Trace:
#0 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2015-03-15 16:44:01 Error: [MissingControllerException] Controller class CakephpController could not be found.
Exception Attributes: array (
  'class' => 'CakephpController',
  'plugin' => NULL,
)
Request URL: /cakephp/blog/cakephp/blog/posts
Stack Trace:
#0 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2015-03-15 16:44:02 Error: [MissingControllerException] Controller class CakephpController could not be found.
Exception Attributes: array (
  'class' => 'CakephpController',
  'plugin' => NULL,
)
Request URL: /cakephp/blog/cakephp/blog/posts
Stack Trace:
#0 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2015-03-15 16:44:02 Error: [MissingControllerException] Controller class CakephpController could not be found.
Exception Attributes: array (
  'class' => 'CakephpController',
  'plugin' => NULL,
)
Request URL: /cakephp/blog/cakephp/blog/posts
Stack Trace:
#0 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2015-03-15 16:44:02 Error: [MissingControllerException] Controller class CakephpController could not be found.
Exception Attributes: array (
  'class' => 'CakephpController',
  'plugin' => NULL,
)
Request URL: /cakephp/blog/cakephp/blog/posts
Stack Trace:
#0 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2015-03-15 16:44:03 Error: [MissingControllerException] Controller class CakephpController could not be found.
Exception Attributes: array (
  'class' => 'CakephpController',
  'plugin' => NULL,
)
Request URL: /cakephp/blog/cakephp/blog/posts
Stack Trace:
#0 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2015-03-15 16:44:15 Error: [MissingControllerException] Controller class CakephpController could not be found.
Exception Attributes: array (
  'class' => 'CakephpController',
  'plugin' => NULL,
)
Request URL: /cakephp/blog/cakephp/blog/posts
Stack Trace:
#0 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2015-03-15 16:45:05 Error: [CakeException] $controller does not implement an isAuthorized() method.
Request URL: /cakephp/blog/
Stack Trace:
#0 /opt/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthorize.php(72): ControllerAuthorize->controller(Object(PagesController))
#1 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(502): BaseAuthorize->__construct(Object(ComponentCollection), Array)
#2 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(464): AuthComponent->constructAuthorize()
#3 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(310): AuthComponent->isAuthorized(Array)
#4 [internal function]: AuthComponent->startup(Object(PagesController))
#5 /opt/cakephp/lib/Cake/Utility/ObjectCollection.php(128): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /opt/cakephp/lib/Cake/Event/CakeEventManager.php(243): call_user_func(Array, Object(CakeEvent))
#8 /opt/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#9 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#10 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PagesController), Object(CakeRequest))
#11 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2015-03-15 16:47:00 Error: [CakeException] $controller does not implement an isAuthorized() method.
Request URL: /cakephp/blog/
Stack Trace:
#0 /opt/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthorize.php(72): ControllerAuthorize->controller(Object(PagesController))
#1 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(502): BaseAuthorize->__construct(Object(ComponentCollection), Array)
#2 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(464): AuthComponent->constructAuthorize()
#3 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(310): AuthComponent->isAuthorized(Array)
#4 [internal function]: AuthComponent->startup(Object(PagesController))
#5 /opt/cakephp/lib/Cake/Utility/ObjectCollection.php(128): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /opt/cakephp/lib/Cake/Event/CakeEventManager.php(243): call_user_func(Array, Object(CakeEvent))
#8 /opt/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#9 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#10 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PagesController), Object(CakeRequest))
#11 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2015-03-15 16:47:01 Error: [CakeException] $controller does not implement an isAuthorized() method.
Request URL: /cakephp/blog/
Stack Trace:
#0 /opt/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthorize.php(72): ControllerAuthorize->controller(Object(PagesController))
#1 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(502): BaseAuthorize->__construct(Object(ComponentCollection), Array)
#2 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(464): AuthComponent->constructAuthorize()
#3 /opt/cakephp/lib/Cake/Controller/Component/AuthComponent.php(310): AuthComponent->isAuthorized(Array)
#4 [internal function]: AuthComponent->startup(Object(PagesController))
#5 /opt/cakephp/lib/Cake/Utility/ObjectCollection.php(128): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /opt/cakephp/lib/Cake/Event/CakeEventManager.php(243): call_user_func(Array, Object(CakeEvent))
#8 /opt/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#9 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#10 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PagesController), Object(CakeRequest))
#11 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2015-03-15 17:12:18 Error: Fatal Error (4): syntax error, unexpected ')' in [/var/www/cakephp/blog/Controller/PostsController.php, line 74]
2015-03-15 17:12:18 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 74)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 74, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 74, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-16 11:16:07 Error: Fatal Error (4): syntax error, unexpected '(', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in [/var/www/cakephp/blog/View/Posts/view.ctp, line 50]
2015-03-16 11:16:07 Error: [FatalErrorException] syntax error, unexpected '(', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 50)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 50, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 50, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-16 11:16:38 Error: Fatal Error (1): Call to a member function create() on a non-object in [/var/www/cakephp/blog/View/Posts/view.ctp, line 45]
2015-03-16 11:16:38 Error: [FatalErrorException] Call to a member function create() on a non-object
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/cakeph...', 45)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/cakeph...', 45, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/cakeph...', 45, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-16 11:16:39 Error: Fatal Error (1): Call to a member function create() on a non-object in [/var/www/cakephp/blog/View/Posts/view.ctp, line 45]
2015-03-16 11:16:39 Error: [FatalErrorException] Call to a member function create() on a non-object
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/cakeph...', 45)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/cakeph...', 45, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/cakeph...', 45, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-16 11:16:39 Error: Fatal Error (1): Call to a member function create() on a non-object in [/var/www/cakephp/blog/View/Posts/view.ctp, line 45]
2015-03-16 11:16:39 Error: [FatalErrorException] Call to a member function create() on a non-object
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/cakeph...', 45)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/cakeph...', 45, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/cakeph...', 45, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-16 11:16:39 Error: Fatal Error (1): Call to a member function create() on a non-object in [/var/www/cakephp/blog/View/Posts/view.ctp, line 45]
2015-03-16 11:16:39 Error: [FatalErrorException] Call to a member function create() on a non-object
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/cakeph...', 45)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/cakeph...', 45, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/cakeph...', 45, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 00:51:17 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '1')
#10 [internal function]: PostsController->view('1')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:51:33 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '1')
#10 [internal function]: PostsController->view('1')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:51:36 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '1')
#10 [internal function]: PostsController->view('1')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:51:37 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '1')
#10 [internal function]: PostsController->view('1')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:51:54 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '1')
#10 [internal function]: PostsController->view('1')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:51:55 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '1')
#10 [internal function]: PostsController->view('1')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:54:16 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '1')
#10 [internal function]: PostsController->view('1')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:54:19 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '1')
#10 [internal function]: PostsController->view('1')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:54:21 Error: [NotFoundException] Invalid post
Request URL: /cakephp/blog/posts/view/
Stack Trace:
#0 [internal function]: PostsController->view()
#1 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#2 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#4 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2015-03-17 00:54:31 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 00:58:45 Error: Fatal Error (4): syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [/var/www/cakephp/blog/Model/User.php, line 96]
2015-03-17 00:58:45 Error: [FatalErrorException] syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 96)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 00:58:47 Error: Fatal Error (4): syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [/var/www/cakephp/blog/Model/User.php, line 96]
2015-03-17 00:58:47 Error: [FatalErrorException] syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 96)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 00:58:47 Error: Fatal Error (4): syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [/var/www/cakephp/blog/Model/User.php, line 96]
2015-03-17 00:58:47 Error: [FatalErrorException] syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 96)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 00:58:47 Error: Fatal Error (4): syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [/var/www/cakephp/blog/Model/User.php, line 96]
2015-03-17 00:58:47 Error: [FatalErrorException] syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 96)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 00:58:47 Error: Fatal Error (4): syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [/var/www/cakephp/blog/Model/User.php, line 96]
2015-03-17 00:58:47 Error: [FatalErrorException] syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 96)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 00:58:48 Error: Fatal Error (4): syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [/var/www/cakephp/blog/Model/User.php, line 96]
2015-03-17 00:58:48 Error: [FatalErrorException] syntax error, unexpected ''' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 96)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 96, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 00:59:12 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:00:54 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:00:54 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:00:55 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:00:55 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:00:55 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:00:55 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:00:56 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:00:56 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(Post), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(Post), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(Post), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /opt/cakephp/lib/Cake/Model/Model.php(1630): Model->find('first', Array)
#9 /var/www/cakephp/blog/Controller/PostsController.php(51): Model->read(NULL, '2')
#10 [internal function]: PostsController->view('2')
#11 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#13 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#14 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2015-03-17 01:05:13 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:13 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:05:14 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:14 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:05:15 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:15 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:05:15 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:15 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:05:15 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:15 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:05:15 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:15 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:05:15 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:15 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:05:15 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:15 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:05:16 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [/var/www/cakephp/blog/View/Posts/view.ctp, line 56]
2015-03-17 01:05:16 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/blog/posts/view/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/cakeph...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/cakeph...', 56, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-17 01:06:38 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Comment.comment_id' in 'field list'
Request URL: /cakephp/blog/users/view/1
Stack Trace:
#0 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `Comment...', Array)
#2 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(668): DboSource->execute('SELECT `Comment...', Array, Array)
#3 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1444): DboSource->fetchAll('SELECT `Comment...', false)
#4 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1278): DboSource->_fetchHasMany(Object(User), 'SELECT `Comment...', Array)
#5 /opt/cakephp/lib/Cake/Model/Datasource/DboSource.php(1137): DboSource->queryAssociation(Object(User), Object(Comment), 'hasMany', 'Comment', Array, Array, true, Array, 0, Array)
#6 /opt/cakephp/lib/Cake/Model/Model.php(3022): DboSource->read(Object(User), Array)
#7 /opt/cakephp/lib/Cake/Model/Model.php(2994): Model->_readDataSource('first', Array)
#8 /var/www/cakephp/blog/Controller/UsersController.php(45): Model->find('first', Array)
#9 [internal function]: UsersController->view('1')
#10 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#11 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#12 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#13 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2015-03-18 17:25:26 Error: [MissingActionException] Action UsersController::login() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'login',
)
Request URL: /cakephp/blog/users/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#2 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2015-03-18 17:27:20 Error: [MissingActionException] Action UsersController::login() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'login',
)
Request URL: /cakephp/blog/users/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#2 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2015-03-18 17:28:50 Error: [MissingActionException] Action UsersController::login() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'login',
)
Request URL: /cakephp/blog/users/login
Stack Trace:
#0 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#2 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2015-03-21 10:53:54 Error: [MissingComponentException] Component class ToolbarComponent could not be found.
Exception Attributes: array (
  'class' => 'ToolbarComponent',
  'plugin' => 'DebugKit',
)
Request URL: /cakephp/blog/posts
Stack Trace:
#0 /opt/cakephp/lib/Cake/Controller/ComponentCollection.php(53): ComponentCollection->load('DebugKit.Toolba...', Array)
#1 /opt/cakephp/lib/Cake/Controller/Controller.php(643): ComponentCollection->init(Object(PostsController))
#2 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(188): Controller->constructClasses()
#3 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#4 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2015-03-21 10:57:16 Error: Fatal Error (1): Call to a member function setFlash() on a non-object in [/var/www/cakephp/blog/Controller/PostsController.php, line 108]
2015-03-21 10:57:16 Error: [FatalErrorException] Call to a member function setFlash() on a non-object
Request URL: /cakephp/blog/posts/delete/2
Stack Trace:
#0 /opt/cakephp/lib/Cake/Error/ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/cakeph...', 108)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/cakeph...', 108, Array)
#2 /opt/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/cakeph...', 108, Array)
#3 /opt/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2015-03-21 10:59:52 Error: [NotFoundException] Invalid post
Request URL: /cakephp/blog/posts/delete/2
Stack Trace:
#0 [internal function]: PostsController->delete('2')
#1 /opt/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#2 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 /opt/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#4 /var/www/cakephp/blog/webroot/index.php(109): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
